### PR TITLE
Fixed file path issue on Windows

### LINKED ISSUE #125: Duplicate suggestions
image

Noteworthy is that one `Name` suggestion shows docs, the other doesn't. Looks like one is inferred from the existing `Name` tag.
Expected would be 0 suggestions for `Name`, because `Name` must only appear once.

Schema is here: https://github.com/felixfbecker/PowerShellXSD/blob/master/Types.xsd

Relevant section:
```xml
<xs:complexType name="Type">
    <xs:sequence>
      <xs:element name="Name" type="xs:string">
        <xs:annotation>
          <xs:documentation>A pair of `&lt;Name&gt;` tags that enclose the name of the affected .NET Framework type.</xs:documentation>
        </xs:annotation>
      </xs:element>
      <xs:element minOccurs="0" name="Members" type="Members" />
      <xs:element minOccurs="0" name="TypeConverter" type="TypeConverter" />
      <xs:element minOccurs="0" name="TypeAdapter" type="TypeAdapter" />
    </xs:sequence>
  </xs:complexType>
```

### PR DESCRIPTION
**Pull request by NikolasKomonen**
Fixes https://github.com/redhat-developer/vscode-xml/issues/125

### CODE REVIEW

File: org.eclipse.lsp4xml/src/test/java/org/eclipse/lsp4xml/utils/ProjectUtils.java [hunk 1]
```diff
@@ -24,6 +24,7 @@
 	 */
 	public static Path getProjectDirectory() {
 		String currPath = ProjectUtils.class.getClassLoader().getResource("").getFile();
+		currPath = FilesUtils.normalizePath(currPath);
```
Comment (fbricon) [inline org.eclipse.lsp4xml/src/test/java/org/eclipse/lsp4xml/utils/ProjectUtils.java]: why is this necessary?

Comment (NikolasKomonen) [reply org.eclipse.lsp4xml/src/test/java/org/eclipse/lsp4xml/utils/ProjectUtils.java]: On Windows the path looking like "\C:\Users\..." was being used and it broke, "\C:\Users\..." the leading '\' broke it because it is not valid in Windows. normalizePath() checks for that.

File: org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/utils/FilesUtils.java [hunk 2]
```diff
@@ -62,15 +66,26 @@ public static void resetDeployPath() {
...
-		if(pathString != null && !pathString.isEmpty()) {
-			pathString = pathString.replaceFirst("^~", System.getProperty("user.home"));
-			Path p = Paths.get(pathString);
-			pathString = p.normalize().toString();
-			return pathString;
-		}
-		return null;
+		
+		if (pathString != null && !pathString.isEmpty()) {
+            if (pathString.indexOf("~") == 0) {
```
Comment (fbricon) [inline org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/utils/FilesUtils.java]: indentation is hosed

File: org.eclipse.lsp4xml/src/test/java/org/eclipse/lsp4xml/utils/ProjectUtils.java [hunk 3]
```diff
@@ -24,6 +24,7 @@
 	 */
 	public static Path getProjectDirectory() {
 		String currPath = ProjectUtils.class.getClassLoader().getResource("").getFile();
```
Comment (fbricon) [inline org.eclipse.lsp4xml/src/test/java/org/eclipse/lsp4xml/utils/ProjectUtils.java]:
```java
String currPath = new File(ProjectUtils.class.getClassLoader().getResource("").getPath()).toString();
```
still ugly, but should remove the leading `/`

File: org.eclipse.lsp4xml/src/test/java/org/eclipse/lsp4xml/utils/FilesUtilsTest.java [hunk 4]
```diff
@@ -27,18 +28,19 @@
...
 	}
 
 	@Test
 	public void normalizePathTest() {
-		assertEquals(System.getProperty("user.home") + "/Test/Folder", FilesUtils.normalizePath("~/Test/Folder"));
-		assertEquals("/Test/~/Folder", FilesUtils.normalizePath("/Test/~/Folder"));
-		assertEquals("~/Test/Folder", FilesUtils.normalizePath("./~/Test/Folder"));
-		assertEquals("/Folder", FilesUtils.normalizePath("/Test/../Folder"));
+		assertEquals(Paths.get(System.getProperty("user.home"), "Test", "Folder").toString(), FilesUtils.normalizePath("~/Test/Folder"));
+		assertEquals(Paths.get(separator, "Test", "~", "Folder").toString(), FilesUtils.normalizePath("/Test/~/Folder"));
```
Comment (NikolasKomonen) [inline org.eclipse.lsp4xml/src/test/java/org/eclipse/lsp4xml/utils/FilesUtilsTest.java]: @fbricon This test was failing for me on Windows, an extra '\' was inserted at the beginning of the output of Paths.get(...) could you check if you have the same issue

**Comment by NikolasKomonen** [comment]
@fbricon updated
